include/openssl/cpu.h

```diff
@@ -92,15 +92,6 @@ extern "C" {
 // Note: the CPUID bits are pre-adjusted for the OSXSAVE bit and the YMM and XMM
 // bits in XCR0, so it is not necessary to check those.
 extern uint32_t OPENSSL_ia32cap_P[4];
-
-#if defined(BORINGSSL_FIPS) && !defined(BORINGSSL_SHARED_LIBRARY)
-const uint32_t *OPENSSL_ia32cap_get(void);
-#else
-OPENSSL_INLINE const uint32_t *OPENSSL_ia32cap_get(void) {
```

Comment:
Related to my comment: I wonder if we wanted to keep this capability for non-fips buids.

Reply:
all calls to this function were inside the fipsmodule, these calls are now changed to direct access to the variable. So the function is not used anywhere anymore.